lib/after_midnight.ex

```diff
@@ -0,0 +1,15 @@
+defmodule BabySitter.AfterMidnight do
+  @after_midnight_rate 16
+
+  def calculate_midnight_pay(end_time) do
+    calculate_after_midnight_hours(end_time) * @after_midnight_rate
+  end
+
+  defp calculate_after_midnight_hours(end_time) when rem(end_time, 24) < 5 do
+    rem(end_time, 24)
+  end
+
+  defp calculate_after_midnight_hours(end_time) when rem(end_time, 24) >= 5 do
```

Comment:
Using a [defguard](https://hexdocs.pm/elixir/guards.html#defining-custom-guard-expressions) could remove the duplication here.